src/js/base/model.js

```diff
@@ -36,13 +36,20 @@ export default Backbone.Model.extend(extend({
     // Model fetches default to aborting.
     const fetcher = Backbone.Model.prototype.fetch.call(this, extend({ abort: true }, options));
 
+    this._isFetching = true;
+
     // Resolve with entity if successful
     return fetcher.then(response => {
+      this._isFetching = false;
+
```

Comment:
we should likely process the queue here.  and it's likely `_.each(this._notifications, this._handleMessage);` where `_handleMessage` is after the isFetching return

Reply:
why `, data => { this._handleMessage(data) });` over `, this._handleMessage);`?

Reply:
I was getting an error where `this` is `undefined`, is why I switched to that.

`each(this._notifications, this._handleMessage);` results in that error. But I just tested this and it works:       `each(this._notifications, () => { this._handleMessage });`.

Reply:
Nice that works, updated to that.